db/migrate/20171003153758_add_translation_meta_title_to_product.rb

```diff
@@ -0,0 +1,13 @@
+class AddTranslationMetaTitleToProduct < ActiveRecord::Migration[5.1]
```

Comment:
@bricesanchez please change the `ActiveRecord::Migration[5.1]` to `ActiveRecord::Migration[5.0]` or `ActiveRecord::Migration[4.2]` so tests will pass :)